plugins/indexing/types/message.go

```diff
@@ -1,13 +1,15 @@
 package types
 
 type Message struct {
-	Type string      `json:"type"`
-	Data interface{} `json:"data"`
+	Block *BlockContext `json:"-"`
```

Comment:
why "-" instead of "block"?

Reply:
Because I don’t want the data to be part of the message payload, just the attributes. The `"-"` excludes the attribute from the JSON output if I’m not mistaken.